src/Reflection/ReflectionVariable.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace BetterReflection\Reflection;
+
+use phpDocumentor\Reflection\Type;
+use phpDocumentor\Reflection\Types;
+
+class ReflectionVariable
+{
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var int
+     */
+    private $declaredAt;
+
+    /**
+     * @var $type
+     */
+    private $type;
+
+    /**
+     * @param string $name
+     * @param int $declaredAt
```

Comment:
Well, ideally there would be the range of (characters?) that the variable is available for, but the PHP parser just gives us line numbers ..